ivtest/regress-sv.list

```diff
@@ -600,6 +600,10 @@ sv_foreach2		normal,-g2009		ivltests
 sv_foreach3		normal,-g2009		ivltests
 sv_foreach4		normal,-g2009		ivltests
 sv_foreach5		normal,-g2009		ivltests
+sv_foreach6		normal,-g2009		ivltests
+sv_foreach7		normal,-g2009		ivltests
+sv_foreach8		normal,-g2009		ivltests
```

Comment:
I think it would be handy to include a gold file for the sv_foreach8 test that shows not only that the right number of iterations are taken, but that the indices are correct. The sample output in the PR comment would be a good example of what I'm thinking.

Reply:
Done